packages/testsuite/cypress.config.ts

```diff
@@ -30,47 +29,134 @@ export default defineConfig({
-                        clearInterval(interval);
-                        reject();
-                      }
-                      axios
-                        .post(managementApi, {
-                          operation: "read-attribute",
-                          name: "server-state",
-                        })
-                        .then((response) => {
-                          if ((response as { data: { result: string } }).data.result == "running") {
+                } else {
+                  await wildflyContainer.exec([
+                    `/bin/bash`,
+                    `-c`,
+                    `$JBOSS_HOME/bin/jboss-cli.sh -c --command="/core-service=management/management-interface=http-interface:list-add(name=allowed-origins,value=http://localhost:${
+                      config.env.HAL_CONTAINER_PORT as string
+                    }"`,
+                  ]);
+                  await wildflyContainer.exec([
+                    `/bin/bash`,
+                    `-c`,
+                    `$JBOSS_HOME/bin/jboss-cli.sh -c --command="reload"`,
+                  ]);
+                  await wildflyContainer
+                    .exec([
+                      `/bin/bash`,
+                      `-c`,
+                      `$JBOSS_HOME/bin/jboss-cli.sh -c --command="read-attribute server-state"`,
+                    ])
+                    .then((response) => {
+                      if (response.output.includes("running")) {
+                        const startTime = new Date().getTime();
+                        const interval = setInterval(() => {
+                          if (new Date().getTime() - startTime > 10000) {
                             clearInterval(interval);
-                            resolve(`http://localhost:${wildflyContainer.getMappedPort(9990)}`);
+                            reject();
                           }
-                        })
-                        /* eslint @typescript-eslint/no-empty-function: off */
-                        .catch(() => {});
-                    }, 500);
-                  });
+                        });
+                        clearInterval(interval);
+                        resolve(`http://localhost:9990`);
+                      } else {
+                        console.log("wildfly server is not running");
+                      }
+                    })
+                    /* eslint @typescript-eslint/no-empty-function: off */
+                    .catch(() => {});
+                }
               })
-              .catch((err) => reject(err));
+              .catch((err) => {
+                console.log(err);
+                reject(err);
+              });
+          });
+        },
+        "start:sso:container": ({ name }) => {
+          const ssoContainerBuilder = new GenericContainer(process.env.SSO_IMAGE || "quay.io/keycloak/keycloak:latest")
+            .withName(name as string)
+            .withNetworkMode("host")
```

Comment:
Some documentation and reasoning regarding host network mode would be useful.

Reply:
add doc to interface and use HTML marks if needed